app/controllers/triannon/auth_controller.rb

```diff
@@ -0,0 +1,353 @@
+require_dependency "triannon/application_controller"
+
+module Triannon
+  # Adapted from http://image-auth.iiif.io/api/image/2.1/authentication.html
+  class AuthController < ApplicationController
+    include RdfResponseFormats
+
+    AUTH_EXPIRY  =   60 # seconds
+    TOKEN_EXPIRY = 3600 # seconds
```

Comment:
would it be more appropriate to allow expiry times to be chosen in the triannon.yml  file?